src/Integration.Vsix.UnitTests/SonarLintTagger/TextBufferIssueTrackerTests.cs

```diff
@@ -115,50 +106,24 @@ private void CheckFactoryWasUnregisteredFromDataSource(SnapshotFactory factory,
         #region Triggering analysis tests
 
         [TestMethod]
-        public void WhenTaggerCreated_AnalysisIsRequested()
+        public void WhenCreated_AnalysisIsRequested()
         {
-            // 1. No tagger -> analysis not requested
-            CheckAnalysisWasNotRequested();
-
-            // 2. Add a tagger -> analysis requested
-            using (testSubject.CreateTagger())
-            {
-                mockAnalyzerController.Verify(x => x.ExecuteAnalysis("foo.js", "utf-8", new AnalysisLanguage[] { AnalysisLanguage.Javascript }, It.IsAny<IIssueConsumer>(),
-                    (IAnalyzerOptions)null /* no expecting any options when a new tagger is added */,
-                    It.IsAny<CancellationToken>()), Times.Once);
-            }
+            mockAnalyzerController.Verify(x => x.ExecuteAnalysis("foo.js", "utf-8",
+                new[] {AnalysisLanguage.Javascript}, It.IsAny<IIssueConsumer>(),
+                null /* no expecting any options when a new tagger is added */,
+                It.IsAny<CancellationToken>()), Times.Once);
         }
 
         [TestMethod]
-        public void WhenFileIsSaved_ButNoTaggers_AnalysisIsNotRequested()
+        public void WhenFileIsSaved_AnalysisIsRequested()
         {
-            // Arrange
-            CheckAnalysisWasNotRequested();
-
-            // Act
-            RaiseFileLoadedEvent(mockedJavascriptDocumentFooJs);
-
-            // Assert
-            CheckAnalysisWasNotRequested();
-        }
-
-        [TestMethod]
-        public void WhenFileIsSaved_AnalysisIsRequested_ButOnlyIfATaggerIsRegistered()
-        {
-            // 1. No tagger -> analysis not requested
-            RaiseFileSavedEvent(mockedJavascriptDocumentFooJs);
-            CheckAnalysisWasNotRequested();
-
-            // 2. Add a tagger and raise -> analysis requested
-            var tagger = testSubject.CreateTagger();
             mockAnalyzerController.Invocations.Clear();
 
-
             RaiseFileSavedEvent(mockedJavascriptDocumentFooJs);
             CheckAnalysisWasRequested();
 
-            // 3. Unregister tagger and raise -> analysis not requested
-            tagger.Dispose();
+            // Unregister tagger and raise -> analysis not requested
```

Comment:
Comment is now out of date

Reply:
Changed to `// Dispose and raise -> analysis not requested`